package/packages.yaml

```diff
@@ -47,7 +47,7 @@ local:
         name: Daml.Finance.Interface.Types.Date
         path: main/daml/Daml.Finance.Interface.Types.Date
         base-module: Daml.Finance.Interface.Types.Date
-        increment-version: patch
+        increment-version: major
```

Comment:
I would keep `patch`, as major changes are hopefully the minority, but no strong opinion

Reply:
No strong opinion from my side either. I think I changed it to major after your previous feedback that it was a breaking change, but after yesterday's discussion we might want to reconsider the policy.  I have added it as a discussion point for tomorrow. Maybe we can merge as-is though, as it fits the current policy, and then change if needed?

Reply:
I think yours is a breaking change, hence the bump of the major version. That said, I would leave the default to `patch` otherwise each time you run packell you get a major version bump :)

Reply:
Oh, I see what you mean now. I created a separate PR for that, since this one was auto-merged.